$OpenBSD: patch-configure,v 1.8 2015/07/23 07:33:17 landry Exp $
no Werror, breaks with gcrypt deprecation warnings
dont check for pcap-bpf.h, not installed on openbsd
--- configure.orig	Wed May 27 10:51:50 2015
+++ configure	Wed Jun  3 15:45:37 2015
@@ -17312,7 +17312,7 @@ SAVE_CFLAGS="$CFLAGS"
 # Emulate behavior of src/Makefile.am
 if test "x$GCC" = "xyes"
 then
-	CFLAGS="$CFLAGS -Wall -Werror"
+	CFLAGS="$CFLAGS -Wall"
 fi
 
 { $as_echo "$as_me:${as_lineno-$LINENO}: checking for strtok_r" >&5
@@ -23486,23 +23486,7 @@ fi
 done
 
 fi
-if test "x$with_libpcap" = "xyes"
-then
-	for ac_header in pcap-bpf.h
-do :
-  ac_fn_c_check_header_mongrel "$LINENO" "pcap-bpf.h" "ac_cv_header_pcap_bpf_h" "$ac_includes_default"
-if test "x$ac_cv_header_pcap_bpf_h" = xyes; then :
-  cat >>confdefs.h <<_ACEOF
-#define HAVE_PCAP_BPF_H 1
-_ACEOF
 
-else
-  with_libpcap="no (pcap-bpf.h not found)"
-fi
-
-done
-
-fi
 if test "x$with_libpcap" = "xyes"
 then
 	{ $as_echo "$as_me:${as_lineno-$LINENO}: checking whether libpcap has PCAP_ERROR_IFACE_NOT_UP" >&5
